chore: docs for the new fields from AccessibilityNodeInfo

ankahanets opened the pull request:
Related PR: https://github.com/appium/appium-uiautomator2-server/pull/682

mykola-mokhnach commented on README.md:
```diff
@@ -222,6 +222,23 @@ bounds | The element's visible frame (`[left, top][right, bottom]`) | `[0,0][100
 displayed | Whether the element is visible to the user | 'true'
 contentSize | The dimensions of the element's content area | `{"left": 0, "top":0, "width": 100, "height": 100, "scrollableOffset": 10, "touchPadding": 0}`
 extras | The result of [getExtras](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo#getExtras()). The value includes all key-value pairs as `key=value` separated by a semicolon (`;`). If the value is empty, then only key part ending with the equal sign will be present. | Part of extras in chrome browser:<br> `AccessibilityNodeInfo.roleDescription=;`<br>`Acc ...
+a11y-important | Whether the element originates from a view considered important for accessibility | 'true'
```
Please also mention the minimum API level needed for corresponding properties to exist as well as the meaning of `null` values

ankahanets replied:
Actually, user can not receive null-value because in this case the field just will be absent.

mykola-mokhnach commented:
Please also update package.json with the minimum server version supporting mentioned attributes

ankahanets commented:
> Please also update package.json with the minimum server version supporting mentioned attributes

I've set `"appium-uiautomator2-server": "^7.2.0"` in package.json, is it correct?

mykola-mokhnach commented:
> > Please also update package.json with the minimum server version supporting mentioned attributes
> 
> I've set `"appium-uiautomator2-server": "^7.2.0"` in package.json, is it correct?

yes, thanks

mykola-mokhnach commented:
@ankahanets would you have some time this week to finish the PR?

ankahanets commented:
@mykola-mokhnach Hi, I'm here. What I need to finish? Fix merge conflict or something else?

mykola-mokhnach commented:
> @mykola-mokhnach Hi, I'm here. What I need to finish? Fix merge conflict or something else?

Yes, merge conflicts plus tune some attribute descriptions (e.g. under which circumstances they are nulls/not present in the page source)

ankahanets commented:
@mykola-mokhnach added "Available only for text input fields" for `input-type`, `multiline` and `max-text-length`, and added "Available only if `includeA11yActionsInPageSource` setting is turned on" for the `actions`.

Also added "Available only if `includeExtrasInPageSource` setting is turned on" for `extras` for consistency.
